File: NewsBrowser/Backend/DTOs/NewsDetails.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Backend.DTOs
{
    public class NewsDetails
    {
        public string Id { get; set; }
        public string Url { get; set; }
        public string Title { get; set; }
        public string Author { get; set; }
        public string Text { get; set; }
        public List<string> Tags { get; set; }
        public string ImageUrl { get; set; }
        public string Language { get; set; }
        public string SiteSection { get; set; }
        public string TitleSection { get; set; }
    }
}

File: NewsBrowser/NewsBrowser/ClientApp/src/app/models/NewsDetails.ts
import { Tag } from "./tag";

export interface NewsDetails {
  id: string;
  url: string;
  title: string;
  author: string;
  text: string;
  tags: Tag[];
  imageUrl: string;
  language: string;
  siteSection: string;
  titleSection: string;
}

export const EMPTY_NEWS_DETAILS = {
  id: "",
  url: "",
  title: "",
  author: "",
  text: "",
  tags: [],
  imageUrl: "",
  language: "",
  siteSection: "",
  titleSection: "",
}
